pkg/controller/namespace/namespace_controller_utils.go

```diff
@@ -184,6 +184,12 @@ func deleteAllContent(kubeClient clientset.Interface, versions *unversioned.APIV
 				return estimate, err
 			}
 		}
+		if containsResource(resources, "replicasets") {
```

Comment:
This if statement will return false so ` deleteReplicaSets` will never run, have no idea what's going wrong.